modules/ROOT/pages/changelogs.adoc

```diff
@@ -48,6 +78,7 @@ For more information, see xref:notifications/index.adoc#gqlstatus-notification-o
 Neo.ClientNotification.Cluster.RequestedTopologyMatchedCurrentTopology
 Neo.ClientNotification.Schema.IndexOrConstraintAlreadyExists
 Neo.ClientNotification.Schema.IndexOrConstraintDoesNotExist
+Neo.ClientError.Statement.InvalidTargetDatabaseError
```

Comment:
This one is not documented, needs to be added to the "List of Neo4j error codes" page also